### DISCUSSION TITLE
add "include image" option into ngalert

### DISCUSSION THREAD

**Discussion by nklym** [grafana/grafana#38003] [Alerting]
**ngalert** is a good continuation of the alerting in grafana, but getting  alerts without images is inconvenient. 

In my opinion "include image" is a very useful function. But it still unavailables in grafana 8.1.1 with ngalert

![image](https://user-images.githubusercontent.com/58990524/129872841-17e92687-ea78-4165-b736-f8e7a41e2e78.png)

**Comment by gotjosh** [comment]
Locking in favour of https://github.com/grafana/grafana/discussions/38030
